awx/api/generics.py

```diff
@@ -205,6 +205,9 @@ def finalize_response(self, request, response, *args, **kwargs):
             response['X-API-Query-Count'] = len(q_times)
             response['X-API-Query-Time'] = '%0.3fs' % sum(q_times)
 
+        if getattr(self, 'deprecated', False):
+            response['Warning'] = '299 awx "This resource has been deprecated and will be removed in a future release."'  # noqa
```

Comment:
Oh! this is a server-side change. What's going on with the "229 awx" here?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Warning

Reply:
Oh that's super cool, but since Chrome is said to support this, does that affect the display in the browser?

Reply:
It doesn't look to me like Chrome (or Safari, for that matter) displays anything special in the browser interface.